### PR TITLE
docs: "Frequently Asked Questions" page

### PR DESCRIPTION
**Pull request by iuioiua**
First pass. Other questions that should be answered:
1. Why is the Standard Library versioned differently to the Deno runtime?
2. What are the compatibility guarantees for the Standard Library?

@kt3k, are you able to handle these?

### CODE REVIEW

File: .github/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+We recommend using the latest version of a given package.
+
+Thanks to JSR, you can constrain dependency versions by defining the SemVer
+range in the import specifier.
+
+E.g. If the latest major version of a package is 1, it should be imported as
+follows:
+
+```ts, ignore
+import { bar } from "jsr:@std/foo@1/bar";
```
Comment (kt3k) [inline .github/FAQ.md:52]: `deno add <pkg>` adds import map entry with the form of `"@std/foo": "jsr:@std/foo@^1.2.3"`. The above example seems slightly inconsistent with it to me

File: .github/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+1. It's been moved to a more appropriate location. E.g. To another package.
+1. It's been renamed more appropriate. E.g. To a name that more clearly
+   describes its behavior.
+1. It's been deemed no longer fitting to the Standard Library.
+
+## When is a deprecated API removed?
+
+In almost all cases, a deprecated API is removed in the next major version after
+deprecation. Rarely, a deprecated API is removed in the a minor version as a
+small breaking change. This is to minimize breaking changes for users.
```
Comment (kt3k) [inline .github/FAQ.md:82]: > Rarely, a deprecated API is removed in the a minor version as a
small breaking change.

I personally doubt we would ever do this intentionally. Also I find the term 'small breaking change' strange. What makes a breaking change 'small'?

**Comment by kt3k** [comment]
> .github/FAQ.md

I wonder this is a right place to document this information. How about adding these sections in "Standard Library" page of [deno-docs](https://github.com/denoland/deno-docs) https://docs.deno.com/runtime/manual/basics/standard_library ?

File: .github/FAQ.md (hunk 2 above)
Comment (iuioiua) [reply .github/FAQ.md]: Small breaking changes are those with minimal impact. Either way, I've removed that section now.

File: .github/FAQ.md (hunk 1 above)
Comment (iuioiua) [reply .github/FAQ.md]: Sorry, what are you suggesting? I changed it to `jsr:@std/foo@^1`. How's that?

**Comment by iuioiua** [comment]
> > .github/FAQ.md
> 
> I wonder this is a right place to document this information. How about adding these sections in "Standard Library" page of [deno-docs](https://github.com/denoland/deno-docs) https://docs.deno.com/runtime/manual/basics/standard_library ?

Almost all information on the Standard Library should be contained within this repo. It doesn't sense to disperse this information elsewhere.

**Comment by kt3k** [comment]
> Almost all information on the Standard Library should be contained within this repo. It doesn't sense to disperse this information elsewhere.

Maybe this topic is out of scope of this PR, but the issue with `.github/FAQ.md` is that this file doesn't appear anywhere in jsr.io. I think there's high chance that users miss this file.

**Review by kt3k — APPROVED** [review]
LGTM

**Comment by iuioiua** [comment]
> > Almost all information on the Standard Library should be contained within this repo. It doesn't sense to disperse this information elsewhere.
> 
> Maybe this topic is out of scope of this PR, but the issue with `.github/FAQ.md` is that this file doesn't appear anywhere in jsr.io. I think there's high chance that users miss this file.

Hopfully, https://github.com/jsr-io/jsr/issues/507 fixes that. I would be happy to discuss it elsewhere.

**Pull request merged**